### ISSUE TITLE
AOC I2276VW

### ISSUE THREAD

**Issue by KP1533TM2** [ddccontrol/ddccontrol-db#27]
Hello. Attached is the output of ddccontrol that I captured for my new AOC I2276VW monitor.
Weird, it seems that it doesn't work at all, although DDC/CI is enabled in the monitor itself.

[AOC.txt](https://github.com/ddccontrol/ddccontrol-db/files/966662/AOC.txt)

**Comment by kravemir** [comment]
What GPU and drivers do you use? I have successfully used `ddccontrol` on AOC G2260VWQ6 monitor with VESA profile.

I seen somewhere, that some people have trouble getting DDC/CI to work with nvidia GPU.

**Comment by KP1533TM2** [comment]
Yes! It's a GeForce GTX650. I have 2 monitors set up, one over HDMI and this one over DVI-D. The HDMI one is playing nicely with `ddccontrol`, while DVI-D one isn't. I do remember bringing another DVI monitor to my machine to test, and it didn't work too.
Not sure what to do with that now.

**Comment by kravemir** [comment]
Do you have a monitor, which has HDMI and DVI input? Try to test it on both inputs (not simultaneously connected)  to eliminate possibility, that a monitor might be a cause.

**Comment by corna** [comment]
I'm having the same issue and I'd really like to fix it.
Any starting point?

**Comment by larstobi** [comment]
Thank you for the report. Unfortunately, the attached scan does not contain any valid DDC/CI control responses: reading the capabilities fails and the controls list is empty. This does not give us enough information to create a safe monitor profile.

If possible, please run a new scan using a connection where DDC/CI communication is known to work (trying another GPU, driver, cable, or monitor input if necessary), with DDC/CI enabled:

```
LANG=C LC_ALL=C ddccontrol -p -c -d
```

Please attach the complete, unedited output and ask us to reopen this issue. We will be happy to reassess it when a working scan is available.

Closing for now, but a new scan is very welcome.
